pkgs/top-level/all-packages.nix

```diff
@@ -25555,4 +25555,6 @@ in
 
   gortr = callPackage ../servers/gortr {};
 
+  fido2luks = callPackage ../tools/security/fido2luks {};
+
```

Comment:
This should be placed under the appropriate `###########` section -- just place it around the line to `fierce`, that appears to also be in tools/security, if you don't want to look for the proper section yourself :)

Reply:
Fixed in 2aeb4336b715d7cd938a14afe1e38f46d6543732